src/utils/maak_protocol.R

```diff
@@ -0,0 +1,263 @@
+#' @title Create a folder with a markdown template to start working on a new fieldwork protocol
+#'
+#' @description This function will create a new folder based on values that are passed on via the parameters and creates a R-markdown skeleton based on a template file to start working on a new protocol.
+#'
+#' @param titel
+#' @param subtitel
+#' @param auteurs
+#' @param datum
+#' @param reviewers
+#' @param documentbeheerder
+#' @param revisie
+#' @param projectnaam
+#' @param thematiek
+#'
+#' @return A new subfolder beneath src with an Rmarkdown file
+#'
+#' @importFrom rprojroot find_root
+#' @importFrom rprojroot is_git_root
+#' @importFrom whisker whisker.render
+#' @importFrom purrr map_chr
+#' @importFrom stringr str_replace_all
+#' @importFrom assertthat
+#' assert_that
+#' is.string
+#' is.date
+#' @examples
+#' maak_protocol()
+#'
+maak_svp <- function(
+  titel = "Titel",
+  subtitel = "",
+  korte_titel = "korte titel",
+  auteurs = "voornaam1 naam1, voornaam2 naam2, ...",
+  datum = Sys.Date(),
+  reviewers = "voornaam1 naam1, voornaam2 naam2, ...",
+  documentbeheerder = "voornaam naam",
+  revisie = "0.1",
```

Comment:
Thanks for this suggestion. It is very helpful and will apply it. This will however effect version numbering of already existing protocols and I will have to think what we best do about that.

Reply:
I like the idea of linking version number and release. Adding or updating a protocol implies a new release and thus a new version number.

I would suggest to use the date in ISO 8601 format (YYYY-MM-DD) as version number for the release instead of semantic version numbers. And use this version number of the protocol that is added or updated. Other protocols keep their version number.

Reply:
@ThierryO can you expand a little bit on this? Do you mean to abandon semantic version number altogether and just use the date?

Reply:
1. a release will contain the latest version of every protocol
1. a release requires an increasing version number
1. updating a protocol will trigger a new release thus a new version number
1. a link between protocol version and release version seems practical
1. updating only the version number of a protocol is not done

my suggestion is: protocol version = release version in which the protocol is added / updated

## Using semantic version numbers

- v1.0 added proctocol A
    - protocol A 1.0
- v2.0 added protocol B
    - protocol A 1.0
    - protocol B 2.0
- v2.1 minor update protocol A
    - protocol A 2.1
    - protocol B 2.0
- v3.0 major update protocol B
    - protocol A 2.1
    - protocol B 3.0
- v4.0 added protocol C
    - protocol A 2.1
    - protocol B 3.0
    - protocol C 4.0
- v4.1 minor update protocol B
    - protocol A 2.1
    - protocol B 4.1
    - protocol C 4.0

This doesn't make much sence IMHO.

## Using ISO 8601 date

- 2020-03-16 added proctocol A
    - protocol A 2020-03-16
- 2020-03-17 added protocol B
    - protocol A 2020-03-16
    - protocol B 2020-03-17
- 2020-04-15 minor update protocol A
    - protocol A 2020-04-15
    - protocol B 2020-03-17
- 2020-07-19 major update protocol B
    - protocol A 2020-04-15
    - protocol B 2020-07-19
- 2021-01-10 added protocol C
    - protocol A 2020-04-15
    - protocol B 2020-07-19
    - protocol C 2021-01-10 
- 2021-01-10b minor update protocol B
    - protocol A 2020-04-15
    - protocol B 2021-01-10b
    - protocol C 2021-01-10 
 
the version number of a protocol indicates its age

Reply:
@thierryO I think in both cases you don't avoid that a tag or release is named after the version of one specific protocol, leaving other protocols at their previous version. For that aspect I don't see a difference.

In both cases, I'd suggest uniform naming of tags and releases, e.g.
- `svp-403-nl-1.0.0`
- `svp-403-nl-2020-04-15` or `2020-04-15-svp-403-nl`

That should more easily allow to find out which versions were released of each protocol, and access them.

It may indeed be interesting to see what is the age of a protocol; on the other hand you loose some 'version hierarchy' e.g. 0.2.1 is a hotfix of 0.2.0, while 0.3.0 adds more feature(s) and 1.0.0 is a major overhaul. Date is also available within the protocol; further I'm not so sure it is that important to know it from the name. Is maybe also a matter of preference.

Perhaps more important is to make sure there's no misunderstanding about your intention with:

> 3. updating a protocol will trigger a new release thus a new version number

I would not release each commit, if that's your point (I think not?). Releases are deliberately chosen specific commits. That brings us to the question: how to state the version number within the protocol Rmd, in non-release commits (i.e. development) in order to prevent confusion with releases? In the semantic versioning, it is done with `.9000`, not sure how you propose to do that with the date formatted ones.